test/metabase/driver/testing/interactive.clj

```diff
@@ -0,0 +1,112 @@
+(ns metabase.driver.testing.interactive
+  "Tools for interactively testing drivers (especially third party)"
+  (:require [clojure.string :as str]
+            [metabase.driver :as driver]
+            [metabase.models :refer [Database]]
+            [metabase.test :as mt]
+            [metabase.test-runner :as test-runner]
+            [metabase.test.util :as tu]
+            [metabuild-common.input :as input]))
+
+(def ^:private ^:const nil-entry
+  "Line that the user can enter to designate a nil value (needed because [[read-line]] apparently doesn't return
+  a blank line when run under nREPL).
+  Remove if possible"
+  "NIL")
```

Comment:
Dumb question but why can't you just have someone type `nil` or press enter without typing anything to indicate `nil`

Reply:
It seems that `read-line` from nREPL (at least for me) never actually returns on a blank input.  Didn't dig too far into why that is.  That's the crux of why I had to set up a "marker" `nil` value here.  But I could definitely make it lowercase.